store/store.go

```diff
@@ -273,10 +223,23 @@ func (s *Store) Open(enableSingle bool) error {
 	}
 
 	// Instantiate the Raft system.
-	ra, err := raft.NewRaft(config, s, logStore, logStore, snapshots, s.peerStore, transport)
+	ra, err := raft.NewRaft(config, s, logStore, logStore, snapshots, transport)
 	if err != nil {
 		return fmt.Errorf("new raft: %s", err)
 	}
+
+	if enableSingle {
```

Comment:
This no longer seems correct, and needs work. What is the right way to form the cluster?

Reply:
This seems relevant: https://www.consul.io/docs/guides/bootstrapping.html